baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/safety/SafetyPropagationTransfer.java

```diff
@@ -137,6 +137,11 @@
     private static final Matcher<ExpressionTree> TO_STRING =
             MethodMatchers.instanceMethod().anyClass().named("toString").withNoParameters();
 
+    private static final Matcher<ExpressionTree> THROWABLE_GET_MESSAGE = MethodMatchers.instanceMethod()
+            .onDescendantOf(Throwable.class.getName())
+            .named("getMessage")
```

Comment:
we should also treat [`Throwable#getLocalizedMessage()`](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/lang/Throwable.html#getLocalizedMessage()) as unsafe as well

Reply:
Great idea, updated! Let me know if you think of other types/methods which are always unsafe.

`Throwable.toString()` is unsafe as well, but I'd like to handle that differently later on in a way that captures string covnersion e.g. `"exception: " + myException;`